include/envoy/json/json_object.h

```diff
@@ -133,6 +133,13 @@ class Object {
    * @param name supplies the key name to lookup.
    */
   virtual bool hasObject(const std::string& name) const PURE;
+
+  /**
+   * Validates JSON object against passed in schema.
+   * @param schema supplies the schema in string format. A Json::Exception will be thrown, if
```

Comment:
"A Json::Exception will be thrown if the JSON object doesn't conform to the supplied schema or the schema itself is not valid." (no comma after "will be thrown")